src/languages/LanguagesModel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+require_once 'src/languages/Language.php';
+
+// LanguagesModel, to handle database queries for the languages table
+class LanguagesModel
+{
+    public PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function getLanguageById(int $id): Language 
+    {
+        $query = $this->db->prepare('SELECT `id`, `name` FROM `languages` WHERE `id` = :id;');
+        $query->bindParam(':id', $id);
+        $query->execute();
+        $language = $query->fetch();
+
+        $languageObject = new Language(
+            $language['id'], 
+            $language['name'], 
+        );
+        return $languageObject;
```

Comment:
You should handle failure here - if you ask fo ran id that doesn't exist, then `fetch()` will return `false`

Reply:
Weirdly this function isn't being used from what I can see - I suspect I set it up to test, and then getAllLanguages() got used much more.

Perhaps setting it up with a Language|Bool return would be sufficient for now.

Reply:
Ok, I've implemented that, so it catches an invalid language_id and returns false, instead of trying to create and return a new Language object.